lib/_http_outgoing.js

```diff
@@ -673,6 +675,22 @@ OutgoingMessage.prototype.setHeader = function setHeader(name, value) {
   return this;
 };
 
+OutgoingMessage.prototype.setHeaders = function setHeaders(headers) {
+  if (this._header) {
+    throw new ERR_HTTP_HEADERS_SENT('set');
+  }
+
+  if (headers instanceof Headers === false) {
+    throw new ERR_INVALID_ARG_TYPE('headers', 'Headers', headers);
+  }
+
+  for (const key of headers.keys()) {
```

Comment:
Can't you just use `for(const [key, value] of headers.entries() {` here?

Reply:
There is a linting rule against array destructuring

Reply:
I could do

`for(const entry of headers.entries()) {
 this.setHeader(entry[0], entry[1]);
}
`

ugly

Reply:
I see. Nevermind then.